src/components/EditorBar/EditorBar.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, ReactNode } from 'react';
+import { Container, ConfirmButton, ChildrenWrapper } from './EditorBar.styles';
+import moment from 'moment';
+import { useTranslation } from 'react-i18next';
+
+import SquareButton from '../shared/buttons/SubmitButton';
+
+type Props = {
+  onFinishEdit: () => void;
+  lastSaveTime: number | null;
+  isSaving: boolean;
+  children?: ReactNode;
+};
+
+const EditorBar: FC<Props> = ({
+  onFinishEdit,
+  lastSaveTime,
+  isSaving,
+  children,
+}) => {
+  const { t } = useTranslation();
+  const timeAgo =
+    lastSaveTime &&
+    `${t('editor_bar.saved')} ${moment(lastSaveTime).fromNow()}`;
+
+  return (
+    <Container>
+      <ChildrenWrapper>{children}</ChildrenWrapper>
+
+      <div className="right">
+        <div className="timeAgo">{isSaving ? 'Is saving...' : timeAgo}</div>
```

Comment:
We avoid using `className` in favor of styled-components - mixing this two ways of creating styles is not a good practice. 

Could You please, create separate two _styled-components_ to handle `right` and `timeAgo` styling.